commercetools-models/src/main/java/io/sphere/sdk/customergroups/CustomerGroup.java

```diff
@@ -47,6 +51,10 @@
     String getName();
 
     @Override
+    @Nullable
```

Comment:
Actually i added the @HasUpdateAction 😅 , but the pull request is still pending thats why i didn't use it yet.